otherlibs/win32unix/symlink.c

```diff
@@ -51,9 +90,16 @@ CAMLprim value unix_symlink(value to_dir, value osource, value odest)
   if (!pCreateSymbolicLink) {
     pCreateSymbolicLink = (LPFN_CREATESYMBOLICLINK)GetProcAddress(GetModuleHandle(L"kernel32"), "CreateSymbolicLinkW");
     no_symlink = !pCreateSymbolicLink;
+
+    if (!no_symlink && IsDeveloperModeEnabled()) {
+      additional_symlink_flags = SYMBOLIC_LINK_FLAG_ALLOW_UNPRIVILEGED_CREATE;
+    }
```

Comment:
Again, FWIW - your previous error demonstrates why the having an "initialised" flag was a bad idea (you forgot to set it... being initialised is a property of a piece of a code, not a variable), this slightly less unclearly expresses the logic:
```suggestion
    if (!(pCreateSymbolicLink = (LPFN_CREATESYMBOLICLINK)GetProcAddress(GetModuleHandle(L"kernel32"), "CreateSymbolicLinkW"))) {
      no_symlink = 1;
    } else if (IsDeveloperModeEnabled()) {
      additional_symlink_flags = SYMBOLIC_LINK_FLAG_ALLOW_UNPRIVILEGED_CREATE;
    }
```